app/src/validators/DateValidator.py

```diff
@@ -0,0 +1,19 @@
+from wtforms import ValidationError
+from datetime import date
+import re
+
+from app.src.utilities.logger import logger
+
+class DateValidator():
+
+    def __call__(self, form, field):
+
+
+        input_date = field.data
+
+        if input_date.year < 1900:
```

Comment:
havent you copied that check from other file?
If yes then remove that code from the other file as it is probably not used anymore

Reply:
done